flame/parallelisms/parallelize_fla.py

```diff
@@ -160,8 +160,8 @@ def apply_tp(
         layer_plan = {
             "attn_norm": SequenceParallel(),
             "attn": prepare_module_input(
-                input_layouts=(Shard(1), None),
-                desired_input_layouts=(Replicate(), None),
+                input_kwarg_layouts={"hidden_states": Shard(1)},
+                desired_input_kwarg_layouts={"hidden_states": Replicate()},
```

Comment:
https://github.com/fla-org/flash-linear-attention/blob/64ecf54f3f763cbec7f2a8e2009c700064eb23b4/fla/models/transformer/modeling_transformer.py#L123-L130
<img width="629" alt="image" src="https://github.com/user-attachments/assets/bd6c0a31-7ffd-4615-b963-1c33be7b2fd4" />

We are calling the `attn` module with kwargs, not args, so we need to use input_kwarg_layouts and desired_input_kwarg_layouts here